contracts/oracles/Oracle.sol

```diff
@@ -133,9 +133,9 @@ contract Oracle is Ownable, ReentrancyGuard {
         require(token.allowance(msg.sender, address(this)) >= _botAmount);
 
         Participant storage participant = participants[msg.sender];
-        participant.stakeContributed = participant.stakeContributed.add(_botAmount);
-        participant.resultIndex = _eventResultIndex;
         participant.didSetResult = true;
+        participant.resultIndex = _eventResultIndex;
+        participant.stakeContributed = _botAmount;
 
         eventResults[_eventResultIndex].votedBalance = eventResults[_eventResultIndex].votedBalance.add(_botAmount);
```

Comment:
votedBalance.add(_botAmount) and totalStakeContributed.add(_botAmount) need to be evaluation too, same as stakeContributed.

Reply:
Above is not true. Those are not at participant level.